pkg/destroy/gcp/network.go

```diff
@@ -434,41 +531,56 @@ func (o *ClusterUninstaller) listNetworks() ([]nameAndURL, error) {
 }
 
 func (o *ClusterUninstaller) deleteNetwork(name string) error {
-	o.Logger.Debugf("Deleting network %s", name)
 	ctx, cancel := o.contextWithTimeout()
 	defer cancel()
-	_, err := o.computeSvc.Networks.Delete(o.ProjectID, name).Context(ctx).Do()
+
+	o.Logger.Debugf("Deleting network %s", name)
+	resp, err := o.computeSvc.Networks.Delete(o.ProjectID, name).RequestId(o.requestID("network", name)).Context(ctx).Do()
 	if err != nil && !isNoOp(err) {
+		o.resetRequestID("network", name)
 		return errors.Wrapf(err, "failed to delete network %s", name)
 	}
+	if resp.Status == "DONE" && isErrorStatus(resp.HttpErrorStatusCode) {
+		o.resetRequestID("network", name)
+		return errors.Errorf("failed to delete network %s with status code: %d", name, resp.HttpErrorStatusCode)
```

Comment:
I think `httpErrorMessage` is more useful?

Reply:
will fix